src/EditorFeatures/Core/FindUsages/FindUsagesHelpers.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.CodeAnalysis.Editor.FindUsages
 {
     internal static class FindUsagesHelpers
     {
+        public static string GetDisplayName(ISymbol symbol)
+            => symbol.IsConstructor() ? symbol.ContainingType.Name : symbol.Name;
```

Comment:
What about finalizers, operators, etc?  We don't already have a helper for this somewhere?